howToTest.md

```diff
@@ -0,0 +1,76 @@
+[Jest](https://jestjs.io/docs/en/expect) main testing framework
+[Enzyme](https://enzymejs.github.io/enzyme) - Helper testing framework for getting elements from the DOM
+
+### Basic settings:
+
+1. Create '**tests**' folder in component's folder
+2. Create <my-component>.spec.tsx
+3. First line must be (it's IMPORTANT) **import 'jsdom-global/register';**
+4. Then import :
+   - import React from 'react';
+   - import { MyCompnent } from '../MyCompnent'; // your components
+
+### Unit testing of component:
+
+1.  import { shallow } from 'enzyme';
+     shallow or mount or render
+     You should read enzyme [API reference](https://enzymejs.github.io/enzyme/docs/api/) to
+
+2. write your test using 'test' or 'it' function:
```

Comment:
You should write here about grouping our tests with `describe()` and how should we name our tests. Because naming the tests is **really** important, consider the following example:

```javascript
describe('Testing the appController', () => {
    it('should add two numbers together', () => {
      
    });

    it('should substract the second number from the first one', () => {
      
    });

    it('should multiply the two numbers together', () => {
        
    });

    it('should divide the first number with the second', () => {
        
    });
});
```

Notice how individual testcase is being read as plain English sentence - this is good naming. Also I suggest to stick with `it()` and do not use `test()` at all.

[Check it out](https://medium.com/javascript-in-plain-english/writing-your-first-unit-test-with-jest-b4953dabb931)

P.S. Also your sentence needs to be capitalized: _2. **W**rite your test using 'test' or 'it' function:_